.tito/custom.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# All rights reserved.
+#
+# Author: Jan Pokorny <jpokorny@redhat.com>
+# (using src/tito/builder/fetch.py from the tito project as a template)
+#
+# This source code is licensed under the GNU Lesser General Public License
+# version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+#
+
```

Comment:
As the file is not a direct part of pacemaker distribution (as also explicitly stated in `.gitattributes` now, thanks for pointing this out), I see no real need to change anything to how the copyright and license are stated as that's just a copy of what's in libqb already.

I added `from future` imports, though, and reformatted for better PEP 8 compliance.